docs/signing-spec.md

```diff
@@ -242,14 +253,96 @@ by = "GitHub Action [https://github.com/exammple.com/hello_repo]"
 signature = "3dd237895ac..."
 key = "3c44..."
 role = "proxy"
+at = 1611960347
 
 # This identity asserts that it has hosted the bindle
 [[signature]]
 by = "Bindle Server [https://bindle.example.com]"
 signature = "2dd237895ac..."
 key = "2c44..."
 role = "host"
+at = 1611960357
 ```
+ Because of the timestamps, we can reconstruct the timeline and see that the key was
+ first signed by the creator, then signed by the proxy, and finally signed by the host.
+
+## Strategies of Key Trust
+
+This section is non-normative, describing how implementations MAY choose to behave.
+
+A Bindle may be signed by any number of keys in any number of roles.
+During verification, which of these keys must be verified?
+
+According to the text above, a creator key MUST be exist in the keyring, and the content must be verified according to this key.
```

Comment:
"a creator or approver key"

Reply:
I completely rewrote this section, so it should be clearer. The tl;dr is that the simplest config is `verify the creator`, while another strategy (one that is more fitting to public internet) is `verify the creator or an approver`. I think I got that clearer this time around.